### ISSUE TITLE
isfix.zip Flagged as Trojan by Chrome & Microsoft Security/Defender

### ISSUE THREAD

**Issue by Kenshiro3rd** [cracyc/installshieldfix#1]
As the title says, isfix.zip Flagged as Trojan by Chrome & Microsoft Security/Defender. Chrome blocks/removed the download and Defender is left "remediation incomplete".

**Comment by cracyc** [comment]
Clearly a false positive from a behavioural scanner.  They probably flag most things that use detours.  There's nothing I can do about it.

**Comment by Kenshiro3rd** [comment]
Oh I figured it was, and I figured it would.... due to what its trying to do (Getting a program to launch another program to run an installer). I was just just giving you a heads up that it was setting them off.

What caught me by surprise is that it set BOTH off at the same time. Chrome beats Windows Security/Defender to the punch in removing it though... and as a result leaves Windows Security/Defender screaming about "remediation incomplete" because "removal failed". XD
